src/app/(fullscreen)/verify/email/page.tsx

```
@@ -69,25 +71,42 @@ export default function EmailStep() {
-                  >
-                    @ {otherDomain}
-                  </button>
-                </li>
+              <ul
+                className="absolute z-10 mt-1.5 w-full overflow-hidden rounded-md bg-gray-900"
+                role="listbox"
+                tabIndex={-1}
+              >
+                {EMAIL_DOMAINS.map((domain) => {
+                  const selected = domain === emailDomain;
+                  return (
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)